gdc_ng_models/models/misc.py

```diff
@@ -16,7 +16,9 @@
 
 class FileReport(Base):
     __tablename__ = 'filereport'
-    id = Column('id', Integer, primary_key=True)
+
+    id_seq = Sequence("filereport_id_seq", metadata=Base.metadata)
```

Comment:
I assume all of these match the actual sequence?

Reply:
Matches the default generated ones, so there's no conflict with existing ones